src/const.py

```diff
@@ -0,0 +1,30 @@
+#!/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+product = 'cortx'
+service_name = 'rgw_setup'
+install_path = f'/opt/seagate/{product}/rgw'
+ceph_conf_path = '/etc/ceph'
+ceph_conf_file = f'{ceph_conf_path}/ceph.conf'
+ceph_conf_tmpl_file = f'{install_path}/conf/ceph.conf'
+ceph_conf_tmpl_url = f'ini://{ceph_conf_tmpl_file}'
+ceph_admin_keyring_file = f'{ceph_conf_path}/ceph.client.admin.keyring'
+ceph_rpms = ['ceph-radosgw', 'ceph-common', 'ceph-base', 'gperftools-libs',
```

Comment:
Can we move this to rpm level dependency instead of hard coding in the code.

Reply:
In post-install we are  validating these rpms are installed or not, hence define this list here.

Reply:
Since we follow image path now (as against rpm) likely the rpms will be installed already at build time.
Keeping in view this can be moved to RPM level dependency so that they can be identified at build time.

Reply:
We have informed all required RPMS dependencies to RE.
Please refer jira comment, here we have captured all dependencies list for cortx-rgw-integration RPM - https://jts.seagate.com/browse/EOS-27792?focusedCommentId=2288052&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2288052

Reply:
These Ceph RPMs are maintained, pulled in and made part of image by RE. Here using this list, RGW mini provisioner will just validate that these are indeed present.